### ISSUE TITLE
Legality of distributed images?

### ISSUE THREAD

**Issue by slonopotamus** [cirruslabs/macos-image-templates#89]
As I see, you're publicly distributing images containing huge parts of macOS via [GitHub Package Registry](https://github.com/cirruslabs/macos-image-templates/pkgs/container/macos-ventura-base). My question is: is it legal at all for you to distribute these and for me as a user to download?

I thought that [macOS EULA](https://www.apple.com/legal/sla/docs/macOSVentura.pdf) doesn't allow redistribution:

> Except as expressly permitted in Section 3, you may not rent, lease, lend, sell, redistribute or sublicense the Apple Software.

Section 3 is kinda hard to interpret, but I didn't notice anything related to "each Lessee must review and agree to be bound by the terms of this" when downloading images via [Tart](https://tart.run/).

So, even if this is legal, it would be great to have an explanation what usage terms cover stuff-that-is-downloaded-via-Tart.

**Comment by fkorotkov** [comment]
Disclaimer, we are not in a position to provide legal consultancy but I can share one observation regarding your concerns about Tart users not being in agreement with the macOS EULA. 

Tart only works on Apple-branded hardware running macOS which indicates that a user already agreed and is in compliance with the terms.

To put some extra credibility in place, Cirrus Labs is a registered developer of virtualization software and we went through a review with Apple. We did not explicitly cover this topic but the review went smoothly and that's how we got allowed to use [com.apple.vm.networking entitlements](https://developer.apple.com/documentation/bundleresources/entitlements/com_apple_vm_networking) after the review to support bridged network.
